client/reader/following-manage/index.jsx

```diff
@@ -123,6 +123,11 @@ class FollowingManage extends Component {
 
 export default connect(
 	( state, ownProps ) => ( {
-		searchResults: getReaderFeedsForQuery( state, ownProps.sitesQuery ) || [],
+		// the feed searching gives different urls than /following/mine.
+		// therefore we decide to omit it and use the feed_URL from the feed/site
+		searchResults: map(
```

Comment:
if we're going to do this, it should be in a memoized selector, not here. This runs every time the redux store updates and would return a new instance of `searchResults` on each run. That would cause the connected component to always rerender...